test/html.js

```diff
@@ -415,26 +415,32 @@ describe('HTML Parser', function () {
 				root.firstChild.getAttribute('alt').should.eql('«Sogno');
 				root.firstChild.rawAttributes.alt.should.eql('&laquo;Sogno');
 			});
-			it('shoud not decode text', function () {
+
+			it('should not decode text from parseHTML()', function () {
 				// https://github.com/taoqf/node-html-parser/issues/33
-				const root = parseHTML(`<html>
-<body>
-<div id='source'>
-&lt;p&gt;
-This content should be enclosed within an escaped p tag&lt;br /&gt;
-&lt;/p&gt;
-</div>
-</body>
-</html>`)
-				root.toString().should.eql(`<html>
-<body>
-<div id='source'>
-&lt;p&gt;
-This content should be enclosed within an escaped p tag&lt;br /&gt;
-&lt;/p&gt;
-</div>
-</body>
-</html>`);
+				const content = `&lt;p&gt; Not a p tag &lt;br /&gt; at all`;
+				const root = parseHTML(`<div>${content}</div>`);
+				root.childNodes.should.have.length(1);
+
+				const divNode = root.firstChild;
+				divNode.childNodes.should.have.length(1);
+
+				const textNode = divNode.firstChild;
+				textNode.rawText.should.eql(content);
+			});
+
+			it(`should decode for node text property`, function () {
+				const encodedText = `My&gt;text`;
+				const decodedText = `My>text`;
+				const root = parseHTML(`<p>${encodedText}</p>`);
+
+				const pNode = root.firstChild;
+				pNode.text.should.eql(decodedText);
+				pNode.rawText.should.eql(encodedText);
+
+				const textNode = pNode.firstChild;
+				textNode.text.should.eql(decodedText);
+				textNode.rawText.should.eql(encodedText);
 			});
```

Comment:
It made sense to keep this test under the encode/decode heading, as this is core functionality.